### PR TITLE
Some README changes

### PR DESCRIPTION
**Pull request by awf**
Update README to point towards nanoDFT as the primary entry point

### CODE REVIEW

**Review by AlexanderMath — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -1,24 +1,45 @@
...
 | [**Training SchNet**](#training-schnet-on-qm1b)
 | [**QM1B dataset**](qm1b/README.md)
 
-[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/ipobmC)
-[![notebook-tests](https://github.com/graphcore-research/pyscf-ipu/actions/workflows/notebooks.yaml/badge.svg)](https://github.com/graphcore-research/pyscf-ipu/actions/workflows/notebooks.yaml)
 
-**Port of PySCF to Graphcore IPU.**
+# PySCF on IPU
+
+PySCF-IPU is built on top of The [PySCF](https://github.com/pyscf) package, porting some of the PySCF algorithms to the Graphcore [IPU](https://www.graphcore.ai/products/ipu).
```
Comment (hatemhelal) [inline README.md:14]: nit:
```suggestion
PySCF-IPU is built on top of the [PySCF](https://github.com/pyscf) package, porting some of the PySCF algorithms to the Graphcore [IPU](https://www.graphcore.ai/products/ipu).

```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,45 @@
...
+The package is under active development, to broaden its scope and applicability.  Current limitations are:
 - Number of atomic orbitals less than 70 `mol.nao_nr() <= 70`.
 - Larger numerical errors due to `np.float32` instead of `np.float64`.
 - Limited support for `jax.grad(.)`
 
+## QuickStart
+
+### For ML dataset generation (SynS & ML Workshop 2023)
+To generate datasets based on the paper __Repurposing Density Functional Theory to Suit Deep Learning__ [Link](https://icml.cc/virtual/2023/workshop/21476#wse-detail-28485) [PDF](https://syns-ml.github.io/2023/assets/papers/17.pdf) presented at the [Syns & ML Workshop, ICML 2023](https://syns-ml.github.io/2023/), the entry point is the notebook [DFT Dataset Generation.ipynb](./notebooks/DFT-dataset-generation.ipynb), and the file [density_functional_theory.py](density_functional_theory.py).
```
Comment (hatemhelal) [inline README.md:26]:
```suggestion
To generate datasets based on the paper __Repurposing Density Functional Theory to Suit Deep Learning__ [Link](https://icml.cc/virtual/2023/workshop/21476#wse-detail-28485) [PDF](https://syns-ml.github.io/2023/assets/papers/17.pdf) presented at the [Syns & ML Workshop, ICML 2023](https://syns-ml.github.io/2023/), the entry point is the notebook [DFT Dataset Generation](./notebooks/DFT-dataset-generation.ipynb), and the file [density_functional_theory.py](./density_functional_theory.py).

```

File: README.md [hunk 3]
```diff
@@ -1,24 +1,45 @@
...
+
+To run the notebook on Graphcore IPU hardware on Paperspace:
+
+[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/ipobmC)
+
+### For DFT teaching and learning: nanoDFT
+
+We also provide a lightweight implementation of the SCF algorithm, optimized for readability and hackability, in the notebook [notebooks/nanoDFT-demo](notebooks/nanoDFT-demo.ipynb) and in [nanodft](pyscf_ipu/nanoDFT/README.md)
```
Comment (hatemhelal) [inline README.md:34]:
```suggestion
We also provide a lightweight implementation of the SCF algorithm, optimized for readability and hackability, in the [nanoDFT demo](notebooks/nanoDFT-demo.ipynb) notebook and in [nanodft](pyscf_ipu/nanoDFT/README.md) folder.

```

**Review by hatemhelal — APPROVED** [review]
some minor suggestions but looks good

**Pull request merged**